paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateConfirmationFormScreenshotTest.kt

```diff
@@ -19,32 +18,70 @@ class BacsMandateConfirmationFormScreenshotTest {
-            StripeTheme {
-                BacsMandateConfirmationFormView(
-                    state = BacsMandateConfirmationViewState(
-                        accountNumber = "00012345",
-                        sortCode = "10-88-00",
-                        email = "email@email.com",
-                        nameOnAccount = "John Doe",
-                        payer = resolvableString(R.string.stripe_paymentsheet_bacs_notice_default_payer),
-                        debitGuaranteeAsHtml = resolvableString(
-                            R.string.stripe_paymentsheet_bacs_guarantee_format,
-                            resolvableString(R.string.stripe_paymentsheet_bacs_guarantee_url),
-                            resolvableString(R.string.stripe_paymentsheet_bacs_guarantee)
-                        ),
-                        supportAddressAsHtml = resolvableString(
-                            R.string.stripe_paymentsheet_bacs_support_address_format,
-                            resolvableString(R.string.stripe_paymentsheet_bacs_support_default_address_line_one),
-                            resolvableString(R.string.stripe_paymentsheet_bacs_support_default_address_line_two),
-                            resolvableString(R.string.stripe_paymentsheet_bacs_support_default_email),
-                            resolvableString(R.string.stripe_paymentsheet_bacs_support_default_email)
-                        )
+            BacsMandateConfirmationFormView(
+                state = BacsMandateConfirmationViewState(
+                    accountNumber = "00012345",
+                    sortCode = "10-88-00",
+                    email = "email@email.com",
+                    nameOnAccount = "John Doe",
+                    payer = resolvableString(R.string.stripe_paymentsheet_bacs_notice_default_payer),
+                    debitGuaranteeAsHtml = resolvableString(
+                        R.string.stripe_paymentsheet_bacs_guarantee_format,
+                        resolvableString(R.string.stripe_paymentsheet_bacs_guarantee_url),
+                        resolvableString(R.string.stripe_paymentsheet_bacs_guarantee)
                     ),
-                    viewActionHandler = {}
-                )
-            }
+                    supportAddressAsHtml = resolvableString(
+                        R.string.stripe_paymentsheet_bacs_support_address_format,
+                        resolvableString(R.string.stripe_paymentsheet_bacs_support_default_address_line_one),
+                        resolvableString(R.string.stripe_paymentsheet_bacs_support_default_address_line_two),
+                        resolvableString(R.string.stripe_paymentsheet_bacs_support_default_email),
+                        resolvableString(R.string.stripe_paymentsheet_bacs_support_default_email)
+                    )
+                ),
+                viewActionHandler = {}
+            )
+        }
+    }
+
+    @Test
+    fun testDetails() {
```

Comment:
We probably don't need every variant of this one either, since we already have all the pieces individually.